teorija.tex

```diff
@@ -655,7 +653,7 @@ \section{Realna števila}
   2 &= \frac{p^2}{q^2} \\
   2q^2 &= p^2 \comment{kvadrat $p$ je sodo, torej je tudi $p$ sodo; $p = 2m$} \\
   2q^2 &= \left( 2m \right)^2 \\
-  q^2 &= 2m^2 \comment{kvadrat $q$ je sodo, torej je tudi $q$ sodo; $q = 2n$} \\
```

Comment:
Tole je bilo prav pomoje, saj sem samo vstavil `p` in pokrajšal 2. Šele v naslednji vrstici se vstavi `q = 2n`?